test/components/converters/test_html_to_document.py

```diff
@@ -190,7 +190,7 @@ def test_run_difficult_html(self, test_files_path):
         assert len(result["documents"]) == 1
         assert "Superlinear" in result["documents"][0].content
 
-    @patch("haystack.components.converters.html.extract")
+    @patch("haystack.components.converters.html.extract", return_value="test")
```

Comment:
We now check that `Document.content` is a string or `None`, so we need to pass valid return values in mocks.